### PR TITLE
test(core): add failing test to generatePath, see #6171

### PR DESCRIPTION
**Pull request by sibelius**
this is a failing test to reproduce issue #6171 

not sure how to fix it yet

image

### CODE REVIEW

**Comment by timdorr** [comment]
We need to add the `pretty` option: https://github.com/pillarjs/path-to-regexp/tree/v1.7.0#compile-reverse-path-to-regexp

**Comment by sibelius** [comment]
it works locally, we should rebuild on travis

File: packages/react-router/modules/generatePath.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const compileGenerator = pattern => {
 
   if (cache[pattern]) return cache[pattern];
 
-  const compiledGenerator = pathToRegexp.compile(pattern);
+  const compiledGenerator = pathToRegexp.compile(pattern, { pretty: true });
```
Comment (pshrmn) [inline packages/react-router/modules/generatePath.js]: `pretty` should be passed when calling the compiled function, not when creating it

**Comment by timdorr** [comment]
Whoops! Thanks for that, Paul. Let's just get this merged in because it's the expected behavior.

**Pull request merged**